### ISSUE TITLE
Traccar client hidden cannot start application automatically after reboot

### ISSUE THREAD

**Issue by bernamaxim** [tananaev/traccar-client-android#196]
Application cannot start automatically after reboot at my ACER Liquid Z320 device. 
It use Android version 5.1.1.
After reboot, when I tried to start with call function it doesn’t respond.

**Comment by tananaev** [comment]
I guess you need to check logs to see what's happening.

**Comment by bernamaxim** [comment]
no logs, but please do steps:

1. Start AVD (with traccar-client installed)
<img width="1164" alt="screen shot 2016-05-20 at 2 06 10 am" src="https://cloud.githubusercontent.com/assets/2969479/15406185/86ace232-1e2f-11e6-88eb-d2bfb2b9c95c.png">

Make sure that the “Device Setting” is in the installed application
<img width="546" alt="screen shot 2016-05-20 at 2 07 43 am" src="https://cloud.githubusercontent.com/assets/2969479/15406226/ccb38c90-1e2f-11e6-9047-b58e5955f12d.png”>

2. Please Dial 8722227 (TRACCAR)
<img width="546" alt="screen shot 2016-05-20 at 2 12 26 am" src="https://cloud.githubusercontent.com/assets/2969479/15406353/5d9ff608-1e30-11e6-9bd8-dfd9f47e55d9.png">
It cannot open Traccar Client

**Comment by tananaev** [comment]
It works for me. I need to see logs.

**Comment by maxnet04** [comment]
Allow the phone permissions to the application that will work

**Comment by FarihulR** [comment]
i have the problem liket that. but still not work ! the app still hidden

**Comment by tananaev** [comment]
@FarihulR, are you trying to build a hidden version? If yes, then it's intended behavior that it's removed from launcher. If you want to build regular version, you need to select corresponding flavor.

**Comment by FarihulR** [comment]
@tananaev yes .how ca i do it for select or configure corresponding flavor ?

**Comment by tananaev** [comment]
There is a `Build Variants` panel in Android Studio (usually on the left side at the bottom). Select "regular" version there.

**Comment by FarihulR** [comment]
@tananaev very helpful . that's all working done :+1:
